ci-operator/jobs/red-hat-storage/df-console/red-hat-storage-df-console-main-presubmits.yaml

```diff
@@ -0,0 +1,132 @@
+presubmits:
+  red-hat-storage/df-console:
+    - agent: kubernetes
+      always_run: true
+      branches:
+        - ^main$
+        - ^main-
+      cluster: build01
+      context: ci/prow/lint
+      decorate: true
+      decoration_config:
+        skip_cloning: true
+      labels:
+        ci.openshift.io/generator: prowgen
+        pj-rehearse.openshift.io/can-be-rehearsed: "true"
+      name: pull-ci-red-hat-storage-df-console-main-lint
+      rerun_command: /test lint
+      spec:
+        containers:
+          - args:
+              - --gcs-upload-secret=/secrets/gcs/service-account.json
+              - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+              - --report-credentials-file=/etc/report/credentials
+              - --secret-dir=/secrets/ci-pull-credentials
+              - --target=lint
+            command:
+              - ci-operator
+            image: ci-operator:latest
+            imagePullPolicy: Always
+            name: ""
+            resources:
+              requests:
+                cpu: 10m
+            volumeMounts:
+              - mountPath: /secrets/ci-pull-credentials
+                name: ci-pull-credentials
+                readOnly: true
+              - mountPath: /secrets/gcs
+                name: gcs-credentials
+                readOnly: true
+              - mountPath: /etc/pull-secret
+                name: pull-secret
+                readOnly: true
+              - mountPath: /etc/report
+                name: result-aggregator
+                readOnly: true
+        serviceAccountName: ci-operator
+        volumes:
+          - name: ci-pull-credentials
+            secret:
+              secretName: ci-pull-credentials
+          - name: pull-secret
+            secret:
+              secretName: registry-pull-credentials
+          - name: result-aggregator
+            secret:
+              secretName: result-aggregator
+      trigger: (?m)^/test( | .* )lint,?($|\s.*)
+    - agent: kubernetes
+      always_run: true
+      branches:
+        - ^main$
+        - ^main-
```

Comment:
`master`

Reply:
I assumed we were going to create a new repo (defaults to main), but instead we'll be transferring this over there, reverted.